dev.lynith.Speedtest.json

```diff
@@ -0,0 +1,52 @@
+{
+    "app-id" : "dev.lynith.Speedtest",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "speedtest",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=~/.local/share/"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "build-args" : [],
+        "env" : {
+            "RUST_BACKTRACE" : "1",
+            "RUST_LOG" : "speedtest=debug"
+        }
+    },
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "speedtest",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/LynithDev/speedtest-app",
+                    "tag": "1.0.0"
+                }
```

Comment:
you either need to add the crate sources here, see https://github.com/flatpak/flatpak-builder-tools/tree/master/cargo or use a tarball with vendord crates.